MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -283,13 +281,14 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel(vtkMRMLMarkupsT
   }
   int numberOfMarkups = markups->GetNumberOfFiducials();
 
-  if(numberOfMarkups < 1)
+  if( numberOfMarkups < 1 )
   {
-    if(markupsToModelModuleNode->GetModelNode()!=NULL)
+    if ( markupsToModelModuleNode->GetModelNode() != NULL && markupsToModelModuleNode->GetModelNode()->GetPolyData() != NULL )
     {
+      // TODO: Review this code, see if it is really needed...
```

Comment:
Explain here that we need to set something in the model node's polydata that hides the model but does not generate any warnings.
A small sphere is not ideal, because even if it's hardly visible, it may still throw off "fit-to-screen" function.
Try to set NULL in the polydata and see what happens (check if any warnings are logged).